### ISSUE TITLE
Strange RGBA() conversion

### ISSUE THREAD

**Issue by kaisermann** [ben-eb/gulp-cssnano#29]
Some of my lines that contains a rgba() method, like: **background-color: rgba(whitesmoke,.7);** is being converted to: **background-color:hsla(0,0%,96%,.7);**

Any idea why? There are other places I use the rgba() method and they appear on the final minified css.

Thanks ;)

**Comment by ben-eb** [comment]
Yeah, we convert alpha-transparent rgb to hsl *if* the value is smaller. They are equivalent definitions of the same colour so cssnano will pick the smallest one possible. Note that opaque rgb is converted to hex or keyword depending on whether the value is smaller.

So, with transparency:

```css
h2 {
  color:rgba(199,190,179,.8);
  /* output */
  color:hsla(33,15%,74%,.8);
}
```

But the same hue without:

```css
h2 {
  color:rgba(199,190,179,1);
  /* output */
  color:#c7beb3;
}
```

At the moment this is handled by https://github.com/ben-eb/colormin but could be subject to change for v4.

**Comment by mrondra** [comment]
Hello, 

Is it possible to disable this feature? I have trouble with hsla in IE11, on caniuse.com it shows that it should work, but in real IE11 it is not.

Thanks for reply.
Ondra

**Comment by rodriguezmilton** [comment]
Hi there!
I know this is closed but wanted to check before opening a new issue.
I saw this conversion, and it's great for compression, but a tool I'm using to generate a pdf doesn't support hsla.
I tried to change this conversion by using `legacy: true` at the options of my webpack config (within css-loader), but it does not seem to work.

Any idea on how to prevent this conversion within css-loader? 
TIA!
